src/collective/glossary/interfaces.py

```diff
@@ -21,6 +22,16 @@ class IGlossarySettings(form.Schema):
         default=True,
     )
 
+    enabled_content_types = schema.List(
+        title=_(u'Enabled Content Types'),
+        description=_(u'Only objects of these content types will apply the glossary'),
+        required=False,
+        default=DEFAULT_ENABLED_CONTENT_TYPES,
```

Comment:
@rodfersou by default, all content types but Glossary and Term, should be enabled